inst/default_config.yml

```diff
@@ -6,17 +6,17 @@ default:
     dir_prioritized_loanbooks_and_diagnostics: "path/to/prioritized_loanbooks_and_diagnostics"
     dir_analysis: "path/to/analysis"
   file_names:
-    filename_scenario_tms: "scenarios_2022_tms.csv"
-    filename_scenario_sda: "scenarios_2022_sda.csv"
-    filename_abcd: "abcd.xlsx"
+    filename_scenario_tms: "2023 IEA JRC ISF Auto Fossil Fuel Power (start year 2023).csv"
+    filename_scenario_sda: "2023 CO2 Intensity IEA JRC ISF Steel Cement Aviation (start year 2023).csv"
+    filename_abcd: "2023-02-17_AI_RMI_PACTA for Banks Free dataset_EO_2022Q4.xlsx"
     sheet_abcd: "Company Indicators - PACTA Comp"
   project_parameters:
-    scenario_source: "weo_2022"
+    scenario_source: "weo_2023"
     scenario_select: "nze_2050"
     region_select: "global"
     # normally the start year should correspond with year of the publication of
     # the scenario in use
-    start_year: 2022
+    start_year: 2023
```

Comment:
Is this the only place that it needs to change? 
I guess so, we decided to keep all the cookbook references as is right, in order to not regenerate the sample plots?

Reply:
yeah, I think it's ok if the docs don't use the same values. The values in the docs/cookbook are more informative and in line with the plots, but values here are what is created for the user when using `initialise_default_project()`.

@jacobvjk?

Reply:
we could update it in `config_yml.Rmd` as well. The cookbook itself does not mention the publication year directly

Reply:
overall no strong feelings about this, I would not adjust the cookbook itself and I think the config vignette is a secondary consideration, because it is meant to be more general information

Reply:
I think that 
``` yaml
filename_scenario_sda: "scenarios_2022_ei_p4b.csv"
```
is more informative for the cookbook than
``` yaml
filename_scenario_sda: "2023 CO2 Intensity IEA JRC ISF Steel Cement Aviation (start year 2023).csv"
```
but I could change it to 
``` yaml
filename_scenario_sda: "scenarios_2023_ei_p4b.csv"
```
although that makes it out of sync with the plots in the cookbook... is "scenarios_2023_ei_p4b.csv" preferred?

Reply:
let's keep it as it is, you are right. Being overspecific here looks weird.